tests/openvino/native/models.py

```diff
@@ -93,7 +93,9 @@ def _create_ov_model(self):
         input_2 = opset.parameter([1, 3, 28, 14], name="Input_2")
         multiply = opset.multiply(input_2, 1 / scale, name="Mul")
         add_1 = opset.add(multiply, (-1) * mean, name="Add_1")
-        transpose_fq_input = opset.fake_quantize(add_1, -1, 1, -1, 1, 256, name="Transpose/fq_input_0")
+        # OV bug with element types after fusing preprocessing
+        transpose_fq_input = opset.fake_quantize(add_1,
+             np.float32(-1), np.float32(1), np.float32(-1), np.float32(1), 256, name="Transpose/fq_input_0")
```

Comment:
IDK, but maybe it would be more convenient to have a method that creates dummy fake quantize nodes with the `float32` values to avoid potential bugs & fuses from the backend's side.